homework.md

```diff
@@ -4,38 +4,38 @@
 
 What is 2 + 2?
 
-5
+4
```

Comment:
Good afternoon Rodrigue, I am done reviewing your work, all the answers look correct. Well done.

Reply:
Thank you Sam